integration/src/main/java/scripts/testscripts/WorkspaceLifecycle.java

```diff
@@ -76,11 +76,12 @@ public void doUserJourney(TestUserSpecification testUser, WorkspaceApi workspace
     var createdDate = workspaceDescription.getCreatedDate();
     assertEquals(firstLastUpdatedDate, createdDate);
 
-    var lastUpdatedBy = workspaceDescription.getLastUpdatedBy();
-    assertEquals(USER_EMAIL, lastUpdatedBy);
-
-    var createdBy = workspaceDescription.getCreatedBy();
-    assertEquals(USER_EMAIL, createdBy);
+    // TODO(PF-1752): Change to assert lastUpdatedBy equals to user email once the code changes 735
```

Comment:
I'm curious why you needed to comment it out. This would be for a new workspace, so it would have the activity log, wouldn't it?

Reply:
https://broadinstitute.slack.com/archives/C01JVP0MLLX/p1659370937292819 it's failing in the staging environment where my PR is not in?

Reply:
OK, so I think if you leave the test in, then this will all be sorted as this fix works its way through the system.

Reply:
just to clarify, is this change necessary or should i undo it?

Reply:
I don't think commenting out the test is necessary. However, it does not hurt to leave it commented out now and re-enable it later we have confidence in the liquibase change. Your choice - either is fine with me.